kodi_addon_checker/__main__.py

```diff
@@ -57,16 +61,18 @@ def main():
     parser.add_argument("--version", action="version",
                         version="%(prog)s 0.0.1")
     parser.add_argument("dir", type=dir_type, nargs="*", help="optional add-on or repo directories")
+    parser.add_argument("--branch", help="Branch name on which checker is to run")
```

Comment:
This is required https://docs.python.org/2/library/argparse.html#required

Reply:
I would also suggest to hardcode the choices to the available repos https://docs.python.org/2/library/argparse.html#choices

Reply:
```
    parser.add_argument("--branch", choices=['gotham', 'helix', 'isengard', 'jarvis', 'krypton', 'leia'],
                        required=True, help="Target branch name where the checker will resolve dependencies")
```

Reply:
I have made the suggested changes.